src/uct/ib/rc/verbs/rc_verbs_ep.c

```diff
@@ -673,7 +673,10 @@ UCS_CLASS_INIT_FUNC(uct_rc_verbs_ep_t, const uct_ep_params_t *params)
     uct_rc_txqp_available_set(&self->super.txqp, iface->config.tx_max_wr);
     uct_rc_verbs_txcnt_init(&self->txcnt);
     uct_ib_fence_info_init(&self->fi);
-    uct_rc_verbs_iface_common_prepost_recvs(iface, UINT_MAX);
+    status = uct_rc_verbs_iface_common_prepost_recvs(iface);
+    if (status != UCS_OK) {
+        goto err_qp_cleanup;
+    }
```

Comment:
better to do it on line 671, otherwise the cleanup flow is not fully correct - need also remove that qp from iface array of qps (added on line 672)

Reply:
Fixed.